### ISSUE TITLE
Error in get_best_model()

### ISSUE THREAD

**Issue by 1dan13l** [mike-holcomb/PSOkeras#3]
Hey Mike,
first of all, thanks a lot for the nice algorithm.
I found an error though in get_best_model():

in the line
best_model.compile(loss=self.loss,**optimizer=self.loss**)

should be an optimizer instead of a loss function

best, Ivo

**Comment by SharlotteManganye** [comment]
Thanks, I had the same problem. And your comment helped.
